.github/workflows/push.yml

```diff
@@ -120,6 +120,7 @@ jobs:
       UNAFFECTED: ${{ steps.unaffected.outputs.UNAFFECTED }}
       BUILD_CHUNKS: ${{ steps.build_map.outputs.BUILD_CHUNKS }}
       IMAGES: ${{ steps.deploy_map.outputs.IMAGES }}
+      POLICY_TAG: ${{steps.docker_tags.outputs.POLICY_TAG }}
```

Comment:
Also one thing, consider changing `POLICY_TAG` to something like `LIFECYCLE_TAG`.